flink-runtime/src/main/java/org/apache/flink/runtime/operators/BatchTask.java

```diff
@@ -1264,11 +1264,10 @@ public static void logAndThrowException(Exception ex, AbstractInvokable parent)
 					oe = new OutputEmitter<T>(strategy);
 				}
 				else {
-					final DataDistribution dataDist = config.getOutputDataDistribution(i, cl);
```

Comment:
Can we keep the interface with the `DataDistribution` and simply inject a simple int distribution (1,2,3,...,n)?